spec/unit/footer/footer.spec.js

```diff
@@ -24,9 +24,9 @@ const resizeTo = width =>
   new Promise(resolve => {
```

Comment:
These tests could be simplified a lot more. This doesn't need to return a promise anymore, since it resolves immediately. That said, the tests are currently written assuming this to return a promise, so more changes would be needed to remove the promise.